src/views/Table.vue

```diff
@@ -0,0 +1,187 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div>
+    <CylcObjectMenu />
+    <div class="c-table">
+      <table-component
+        :tasks="tasks"
+        :hoverable="false"
+        :activable="false"
+        :multiple-active="false"
+        :min-depth="1"
+        ref="table0"
+        key="table0"
+      ></table-component>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mixin } from '@/mixins'
+import { datatable } from '@/mixins/tableview'
+import TableComponent from '@/components/cylc/table/Table'
+import { WORKFLOW_TABLE_DELTAS_SUBSCRIPTION } from '@/graphql/queries'
+import Alert from '@/model/Alert.model'
+import CylcObjectMenu from '@/components/cylc/cylcObject/Menu'
+import { mergeWith } from 'lodash'
+import Vue from 'vue'
+
+/**
+* @param {DeltasAdded} data
+* @param {Array} array
+*/
+const applyTableDeltas = (data, array) => {
+  const added = data.added
+  const pruned = data.pruned
+  const updated = data.updated
+  if (added) {
+    if (added.taskProxies) {
```

Comment:
These `if`'s could be combined. Not important, but you can have `if (added && added.taskProxies)` instead of 2 `if`'s. Same for other two `if`'s below.

Reply:
Great suggestion, done!